src/code/z_actor.c

```diff
@@ -456,232 +469,236 @@ void Target_SetColors(TargetContext* targetCtx, Actor* actor, s32 type, PlayStat
     targetCtx->fairyOuter.a = sTatlColorList[type].outer.a;
 }
 
-void Actor_TargetContextInit(TargetContext* targetCtx, Actor* actor, PlayState* play) {
+void Target_Init(TargetContext* targetCtx, Actor* actor, PlayState* play) {
     targetCtx->bgmEnemy = NULL;
-    targetCtx->unk8C = NULL;
+    targetCtx->unk_8C = NULL;
     targetCtx->targetedActor = NULL;
     targetCtx->arrowPointedActor = NULL;
-    targetCtx->unk4B = 0;
-    targetCtx->unk4C = 0;
+    targetCtx->rotation = 0;
+    targetCtx->currentLockOnIndex = 0;
```

Comment:
This was something I was wondering on the message branch, but is it worth naming something `current` when there isn't a `previous` involved? Would just `targetCtx->lockOnIndex = 0;` be fine?